draco/fact_utils.py

```diff
@@ -148,3 +151,17 @@ def answer_set_to_dict(answer_set: Iterable[Symbol], root=ROOT) -> Mapping:
             collector[obj][prop] = collector[obj].get(prop, []) + [child]
 
     return collect_children(root, collector)
+
+
+def facts_to_dict(facts: Specification) -> Mapping:
```

Comment:
I see what you mean.  I added this method to have a one-liner conversion method, just like Draco 1 has. Given that we argue that Draco 2 has a nicer API, I figured it would come across as contradictory in the comparison notebook to have a seemingly more complex conversion flow in Draco 2 than what we have in Draco 1.

But since we would only benefit from this utility in the comparison notebook, we can probably live with this one-liner too:

```python
answer_set_to_dict(next(run_clingo(facts)).answer_set)
```

I'll revert the changes.

Reply:
Short code isn't necessarily better. Composable APIs (imo) are best (with some caveats as it makes sense to have shortcuts for very common calls).